utilities/physics_tendency_updaters.meta

```diff
@@ -0,0 +1,222 @@
+  intent = in
+[ u ]
+  standard_name = x_wind
+  units = m s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = inout
+  state_variable = True
+[ dudt_total ]
+  standard_name = tendency_of_x_wind_due_to_model_physics
+  units = m s-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = inout
+[ dt ]
+  standard_name = timestep_for_physics
+  long_name = time step
+  units = s
+  dimensions = ()
+  type = real | kind = kind_phys
+  intent = in
+[ errcode ]
+  standard_name = ccpp_error_code
+  long_name = Error flag for error handling in CCPP
+  units = flag
+  type = integer
+  dimensions = ()
+  intent = out
+[ errmsg ]
+  standard_name = ccpp_error_message
+  long_name = Error message for error handling in CCPP
+  units = 1
+  type = character | kind = len=512
+  dimensions = ()
+  intent = out
+#####################################################################
+[ccpp-table-properties]
+  name = apply_tendency_of_y_wind
+  type = scheme
+[ccpp-arg-table]
+  name = apply_tendency_of_y_wind_run
+  type = scheme
+[ nz ]
+  standard_name = vertical_layer_dimension
+  long_name = Number of vertical layers
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ dvdt ]
+  standard_name = tendency_of_y_wind
+  units = m s-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ v ]
+  standard_name = y_wind
+  units = m s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = inout
+  state_variable = True
+[ dvdt_total ]
+  standard_name = tendency_of_y_wind_due_to_model_physics
+  units = m s-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = inout
+[ dt ]
+  standard_name = timestep_for_physics
+  long_name = time step
+  units = s
+  dimensions = ()
+  type = real | kind = kind_phys
+  intent = in
+[ errcode ]
+  standard_name = ccpp_error_code
+  long_name = Error flag for error handling in CCPP
+  units = flag
+  type = integer
+  dimensions = ()
+  intent = out
+[ errmsg ]
+  standard_name = ccpp_error_message
+  long_name = Error message for error handling in CCPP
+  units = 1
+  type = character | kind = len=512
+  dimensions = ()
+  intent = out
+#####################################################################
+[ccpp-table-properties]
+  name = apply_heating_rate
+  type = scheme
+[ccpp-arg-table]
+  name = apply_heating_rate_run
+  type = scheme
+[ nz ]
+  standard_name = vertical_layer_dimension
+  long_name = Number of vertical layers
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ heating_rate ]
+  standard_name = heating_rate
+  units = J kg-1 s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
```

Comment:
Based on what @goldy said in #37, I believe there should be a `state_variable=true` here.

Reply:
@nusbaume  - We need a scientist to weigh in on this.  Since my understanding of `heating rate` is that it is replacing state%s, I assumed that it was the state variable.  Is this assumption incorrect?   I really haven't gotten a full grasp of what exactly heating rate is yet.

Reply:
@gold2718 had an opinion about this. We have dry static energy (state%s) already in state. My proposal is that we do NOT change it right now, but have parameterizations that want to update it. That I think is how the model works now? I can dive into it if necessary. Only used in:

eddy_diff_cam.F90
physics_types.F90
vertical_diffusion.F90

(also convect_shallow, unicon and subcol_SILHS)

Reply:
I believe @peverwhee is correct in that this particular variable does NOT need `state_variable=true` here.  The reason is because this quantity isn't actually `state%s`, but just a thermal energy tendency (hence why I requested the name change to `heating_rate` in the first place).  

On top of that I believe this quantity is overwritten by every additional scheme in an SDF that calculates it (similar to something like `ptend_loc` in various CAM physics schemes), which means it also doesn't match my understanding of a state variable from a software design standpoint either.

Of course maybe @gold2718 has a different definition of what a state variable is?

Reply:
I am perfectly happy leaving this as it is, no `state_variable=true` for now.  All of the other `apply` routines had their fields as state variables.  With Jesse's explanation of `heating_rate` being a synonym for us non-science types of `tendency of _temperature`, I agree it is not a state variable.